<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="Func">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>111</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Операции</string>
   </property>
   <widget class="QRadioButton" name="mul">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Умножение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sum">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Сложение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="div">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Деление</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sub">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычитание</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="resbutton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOp1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Число 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOp2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Число 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="operand1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="operand2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="errorlabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>851</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
